src/JuMPModels/space_Shuttle.jl

```diff
@@ -0,0 +1,151 @@
+import Interpolations
+"""
+Space Shuttle Reentry Trajectory Problem:
+    We want to find the optimal trajectory of a space shuttle reentry.
+    The objective is to minimize the angle of attack at the terminal point.
+    The problem is formulated as a JuMP model, and can be found [here](https://jump.dev/JuMP.jl/stable/tutorials/nonlinear/space_shuttle_reentry_trajectory/)
+"""
+function space_Shuttle(integration_rule::String = "rectangular";nh::Int64=503)
```

Comment:
Be consistent in the naming, and avoid capital letters in functions' names: `space_shuttle`